staticjinja/staticjinja.py

```diff
@@ -309,14 +318,16 @@ def get_context(self, template):
         """
         context = {}
         for regex, context_like in self.contexts:
+            # TODO unlink name from the template
```

Comment:
@killjoy1221 Do you have a use case where you aren't using a FileSystemLoader and therefore templates don't have a `.name`? The rest of this library seems to really assume that `.name` exists. You want to open an issue for this?

Reply:
The way the application works currently, name will never be None. But `Template.name` is annotated as `str | None`, so that could cause issues with the type-checker.

Reply:
What is the TODO for then? Seems like this is best type-hinting solution we could do?